ecosystem/platform/server/app/assets/stylesheets/application.tailwind.css

```diff
@@ -11,11 +11,6 @@
   }
 }
 
-html {
-  /* So the sticky header doesn't cover things that are scrolled into view. */
-  scroll-padding-top: 120px;
```

Comment:
Why is this removed? Without it, say you go to an #anchor link, isn't it covered by the HeaderComponent?

Reply:
It's still there, just moved to a utility class: 

https://tailwindcss.com/docs/scroll-padding

https://github.com/aptos-labs/aptos-core/pull/4223/commits/e77ab593aee20a72616036c443a95f9d03aa24c2#diff-cee8494b29ca08559a959775a9bd5dce3345c07d377a7299a8b62b9284c477ad